helm-charts/metal3-deploy/values.yaml

```diff
@@ -19,6 +19,9 @@ global:
   # addresses the DHCP server will manage.
   dhcpRange: 192.168.20.20,192.168.20.80
 
+  # If no dhpHosts set, all mac addresses will be ignored. 
+  dhcpHosts: ""
+
```

Comment:
Doesn't appear the global one is being used either.

Reply:
will remove.